lambdas/backend/routes/catalog/export.js

```diff
@@ -32,9 +31,6 @@ exports.get = async (req, res) => {
       parameters
     }).promise()).body
 
-    const datauri = new Datauri()
-    datauri.format('.zip', resultsBuffer)
```

Comment:
Don't need to format it? Can you explain why?

Reply:
GetExport returns the JSON itself, just as a binary buffer instead of a string or JSON object. There's nothing here *to* format.